packages/framework-core/src/booster-read-model-dispatcher.ts

```diff
@@ -56,7 +56,7 @@ export class BoosterReadModelDispatcher {
 
   private async processFetch(readModelRequest: ReadModelRequestEnvelope): Promise<Array<ReadModelInterface>> {
     const readModelMetadata = this.config.readModels[readModelRequest.typeName]
-    const searcher = Booster.readModel(readModelMetadata.class)
+    const searcher = BoosterApp.readModel(readModelMetadata.class)
```

Comment:
🤔 To be honest, I liked more the name `Booster`. As it acts as the library the user uses in their application (it is not an app, I mean) and I think "BooserApp" adds a bit of confusion. Was there a specific reason to change it?

Reply:
`BoosterApp` is the representation of the current app in the core package. The user will still use the name `Booster`. I would keep working on the refactor proposed in #517 before deciding the final name of the constants in the core package, there's a lot to untangle in there yet!